frontend/workflows/projectCatalog/src/details/quick-links.tsx

```diff
@@ -164,7 +166,7 @@ const QuickLinksCard = ({ linkGroups }: QuickLinksProps) => {
         direction="row"
         alignItems="center"
         spacing={1}
-        style={{ padding: "10px" }}
+        style={{ padding: "10px 10px 0px 10px" }}
```

Comment:
why change this padding?

Reply:
Was to keep parity, I've switched it to margin now which keeps it closer anyways.